### PR TITLE
Add health-check-VIP route returning 200 or 500

### PR DESCRIPTION
**Pull request by wewelll**

### CODE REVIEW

File: source/index.js [hunk 1]
```diff
@@ -121,6 +121,12 @@ module.exports = exports = function(params) {
         return res.send(result);
       });
   });
+  app.get(urn + '-vip', function(req, res) {
+    Promise.resolve(check(params))
```
Comment (cRicateau) [inline source/index.js]: does it really work ? I feel it needs a 'return' before Promise.resolve... @samuelbriole

Comment (wewelll) [reply source/index.js]: Yes it does 😄. I did not invent the Promise.resolve, I copied the one just above.

Comment (wewelll) [reply source/index.js]: @cRicateau Ca te va ?

**Comment by wewelll** [comment]
@daniel-jorge @savoisn

**Comment by daniel-jorge** [comment]
👍

**Pull request merged**